Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM node:10-buster as build-deps
+LABEL maintainer="Penn Labs"
+
+ARG GHOST_API_KEY
+ENV GHOST_API_KEY ${GHOST_API_KEY}
+
+WORKDIR /app/
+
+# Copy project dependencies
+COPY package.json /app/
+COPY yarn.lock /app/
+
+ENV NODE_ENV=production
+
+RUN yarn install --pure-lockfile
```

Comment:
I think you can replace this with just `RUN yarn install --frozen-lockfile --production`

Reply:
not super great semantically. it's the production environment, so the NODE_ENV should be set for both `yarn install` and `yarn build`.